src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -68,8 +68,10 @@
      {:underlay-color (colors/theme-colors colors/neutral-5 colors/neutral-90)
       :style          {:border-radius 16}
       :on-press       #()
-      :on-long-press  #(rf/dispatch [:bottom-sheet/show-sheet
-                                     {:content (drawers/reactions-and-actions message-data context)}])}
+      :on-long-press  #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals